apps/system-e2e/src/tests/islandis/service-portal/smoke/occupational-licenses.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { BrowserContext, expect, test } from '@playwright/test'
+import { icelandicAndNoPopupUrl, urls } from '../../../../support/urls'
+import { session } from '../../../../support/session'
+import { label } from '../../../../support/i18n'
+import { m } from '@island.is/service-portal/core/messages'
+import { disableI18n } from '../../../../support/disablers'
+
+const homeUrl = `${urls.islandisBaseUrl}/minarsidur`
+test.use({ baseURL: urls.islandisBaseUrl })
```

Comment:
Probably not needed since you're just testing the service portal.

Reply:
`test.use` sets the base url you'll be using. However, it's already set in the base config. I.e. `test.use({baseURL:...})` is only useful if your app uses a sub-domain e.g. https://loftbru.island.is